unable to build with -Werror=format-security

The build errors out with `-Werror=format-security` which means I can't release the distro package upgrade:

```
In file included from ../src/anal_ghidra.cpp:4:
/usr/include/libr/r_anal.h:66:48: error: format not a string literal and no format arguments [-Werror=format-sec
urity]
   66 | #define esilprintf(op, fmt, ...) r_strbuf_setf (&op->esil, fmt, ##__VA_ARGS__)
      |                                  ~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../src/anal_ghidra.cpp:1350:9: note: in expansion of macro ‘esilprintf’
 1350 |         esilprintf (anal_op, ss.str()[0] == ','? ss.str().c_str() + 1: ss.str().c_str());
      |         ^~~~~~~~~~
../src/anal_ghidra.cpp: In function ‘int sleigh_op(RAnal*, RAnalOp*, uint64_t, const uint8_t*, int, RAnalOpMask)
’:
../src/anal_ghidra.cpp:1408:46: warning: zero-length gnu_printf format string [-Wformat-zero-length]
 1408 |                         esilprintf (anal_op, "");
      |                                              ^~
/usr/include/libr/r_anal.h:66:60: note: in definition of macro ‘esilprintf’
   66 | #define esilprintf(op, fmt, ...) r_strbuf_setf (&op->esil, fmt, ##__VA_ARGS__)
      |                                                            ^~~
```

That looks good, thanks. I didn't say it needs to be a vuln, just that I can't build with distro compliant flags :cat: 

It gets me compile further, but it seems to have an API incompatibility to radare 5.7.6 :crying_cat_face: 

```
../src/R2LoadImage.cpp: In lambda function:
../src/R2LoadImage.cpp:67:64: error: ‘RAsm’ {aka ‘struct r_asm_t’} has no member named ‘config’
   67 |                                         int inc = (core->rasm->config->bits == 64)? 8: 4;
      |                                                                ^~~~~~
```

--- a/src/anal_ghidra.cpp
+++ b/src/anal_ghidra.cpp
@@ -1347,7 +1347,7 @@ static void sleigh_esil(RAnal *a, RAnalOp *anal_op, ut64 addr, const ut8 *data, 
 	}
 
 	// std::cerr << hex << anal_op->addr << " " << ss.str() << endl;
-	esilprintf (anal_op, ss.str()[0] == ','? ss.str().c_str() + 1: ss.str().c_str());
+	esilprintf (anal_op, "%s", ss.str()[0] == ','? ss.str().c_str() + 1: ss.str().c_str());
 }
 
 #if 0

